webcompat/static/js/lib/issues.js

```diff
@@ -92,14 +92,6 @@ md.renderer.rules.link_open = function(tokens, idx, options, env, self) {
 
 issues.MetaDataView = Backbone.View.extend({
   el: $("#js-Issue-information"),
-  initialize: function() {
```

Comment:
note: this was leftover from the last design. there's nothing to re-render in this view -- it just shows the issue body as-is.